CONTRIBUTING.md

```diff
@@ -90,11 +90,11 @@ This will work on Unix-based systems including Windows Subsystem for Linux.
 
 Once the Python bindings are installed, you can run the system tests for the desired driver. For example:
 
-    pytest src/nidmm/system_tests
+    pytest src/nidmm/system_tests -c generated/nidmm/tox-system_tests.ini
 
 You can also use ``tox`` to run the system tests for the desired driver, *using all installed Python versions*. For example:
 
-    tox -c generated\nidmm\tox-system-tests.ini
+    tox -c generated\nidmm\tox-system_tests.ini
```

Comment:
I hadn't noticed, but these probably should be forward slashes so they work as-is on Linux as well.

Can you confirm that forward slashes don't mess up Windows runs, then fix?

Reply:
Verified and fixed.